learning_observer/learning_observer/auth/social_sso.py

```diff
@@ -83,6 +83,30 @@
         'fetch all text from current rosters.',
     default=False
 )
+pmss.register_field(
+    name="token_uri",
+    type=pmss.pmsstypes.TYPES.string,
+    description="The Canvas OAuth token uri",
+    required=True
+)
+pmss.register_field(
+    name="refresh_token",
+    type=pmss.pmsstypes.TYPES.string,
+    description="The Canvas OAuth refresh token",
+    required=True
+)
+
+pmss.register_field(
```

Comment:
Roster source is already registered in `rosters.py`.
PMSS ought to yell at us when we have duplicate registrations. I created an issue for it in the PMSS repository.

Reply:
Removed duplicate